lib/internal/test_runner/reporter/dot.js

```diff
@@ -23,6 +29,14 @@ module.exports = async function* dot(source) {
     }
   }
   yield '\n';
+
+  if (failedTests.length > 0) {
+    yield `${colors.red}${failedTestSymbol} failing tests:${colors.white}\n`;
```

Comment:
Use uppercase when possible
```
Failed tests:
```

Reply:
Sure, I'll keep that in mind. I assumed it should look similar to the spec reporter, which uses "failing tests:"

Reply:
@nodejs/test_runner WDYT about the formatting?

Reply:
Should I also make it `Failed tests:` for spec reporter?

Reply:
can be in a separate PR